src/build/buildNativefierApp.ts

```diff
@@ -1,7 +1,9 @@
 import * as path from 'path';
 
+import * as asar from 'asar';
 import * as electronGet from '@electron/get';
 import * as electronPackager from 'electron-packager';
+import * as fs from 'fs-extra';
```

Comment:
If using a new third-party dep, you need to declare the extra dep in `package.json`

***BUT*** instead of that, I'd rather avoid the extra dependency (I strive to minimize the amount of third-party deps in Nativefier, to reduce bloat, security vulnerabilities, and upgrade hassle). You don't need an extra package to remove a directory, you can:

- A: use [fs.rmdir](https://nodejs.org/api/fs.html#fs_fs_rmdir_path_options_callback), which is callback-based, so to use it conveniently you'll have to write a little wrapper returning a promise (similar to what `fs-extra` does), like I did for `copyFileOrDir`:
https://github.com/jiahaog/nativefier/blob/f2c816795ac08c724809099ff01d71b022d5a1b4/src/helpers/helpers.ts#L39-L51
- B: simply use [fs.rmDirSync](https://nodejs.org/api/fs.html#fs_fs_rmdirsync_path_options)

Give A. a try, it isn't extra hard and I can help you if you haven't wrapped your head around promises.